### DISCUSSION TITLE
How to use Tailwind in Codesandbox?

### DISCUSSION THREAD

**Discussion by nicooprat** [tailwindlabs/tailwindcss#1576] [General]
Every sandboxes I saw were using the Tailwind built CSS over CDN, but this method doesn't allow to customize it or use PurgeCSS.

When I try to use it the right way (using `vue-cli-plugin-tailwind` in my tests), there are no error but the CSS isn't compiled. I created a repo in Github with this process, then imported it in Codesandbox, and in the `<head>` it just outputs something like this:

```html
<style id="/src/assets/tailwind.css:-css" type="text/css">
@tailwind base;
@tailwind components;
@tailwind utilities;
</style>
```

Here's a simple reproduction test: https://codesandbox.io/s/tailwind-codesandbox-issue-tclfy

The only one sandbox that looks like to use it correctly is this one (using Nuxt) but it's marked as an "Experimental Sandbox", whatever it means: https://codesandbox.io/s/o4vn5pvp7q

Has anyone already succeeded to do this?

Edit: created a discussion in Codesandbox too https://github.com/codesandbox/codesandbox-client/discussions/3918

**Comment by tlgreg** [comment]
I believe the issue is that those are not running in container sandboxes. `node` or `nuxt` templates, for example, should be able to compile custom postcss config.

https://github.com/tailwindcss/playground itself can be used as a codesandbox template, though it seems like csb did some caching and it resolves an older version of tailwindcss (maybe committing the lock file in the repo and updating it with new releases would fix this, needs some testing).
